src/libs/ReportUtils.js

```diff
@@ -752,6 +751,11 @@ function getIcons(report, personalDetails, defaultIcon = null) {
         const parentReport = lodashGet(allReports, [`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID}`]);
         const parentReportAction = ReportActionsUtils.getParentReportAction(report);
 
+        if (!parentReport) {
```

Comment:
How can a report be a thread but does not have a parent report?

Reply:
A report can be a thread but not have access to the parent report. 
So its possible that the thread does not have parent report data in its onyx.

Reply:
Ah so it have a parent report, just not loaded in onyx. Makes sense. Thanks @chiragsalian!
Can you answer my second question (below) it's a bit similar.